GeneratedRegexPatterns/HighConfidenceSecurityModels.json

```diff
@@ -289,6 +301,582 @@
+    "Label": "an Azure Speech Services (Cognitive Services) legacy common annotated security key",
+    "DetectionMetadata": "Identifiable",
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890])(?P<refine>[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{52}JQQJ99[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890][A-L][abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{12}AAAYACOG[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{4})",
+    "Signatures": [
+      "JQQJ"
+    ],
+    "CreatedVersion": "1.18.0",
+    "LastUpdatedVersion": "1.18.0"
+  },
+  {
+    "Id": "SEC101/230",
+    "Name": "AzureSpeechTranslationLegacyCommonAnnotatedSecurityKey",
+    "Label": "an Azure Speech Translation (Cognitive Services) legacy common annotated security key",
+    "DetectionMetadata": "Identifiable",
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890])(?P<refine>[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{52}JQQJ99[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890][A-L][abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{12}AAAZACOG[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{4})",
+    "Signatures": [
+      "JQQJ"
+    ],
+    "CreatedVersion": "1.18.0",
+    "LastUpdatedVersion": "1.18.0"
+  },
+  {
+    "Id": "SEC101/231",
+    "Name": "AzureTextAnalyticsLegacyCommonAnnotatedSecurityKey",
+    "Label": "an Azure Text Analytics (Cognitive Services) legacy common annotated security key",
+    "DetectionMetadata": "Identifiable",
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890])(?P<refine>[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{52}JQQJ99[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890][A-L][abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{12}AAAaACOG[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{4})",
+    "Signatures": [
+      "JQQJ"
+    ],
+    "CreatedVersion": "1.18.0",
+    "LastUpdatedVersion": "1.18.0"
+  },
+  {
+    "Id": "SEC101/232",
+    "Name": "AzureTextTranslationLegacyCommonAnnotatedSecurityKey",
+    "Label": "an Azure Text Translation (Cognitive Services) legacy common annotated security key",
+    "DetectionMetadata": "Identifiable",
+    "Pattern": "(^|[^abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890])(?P<refine>[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{52}JQQJ99[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890][A-L][abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{12}AAAbACOG[abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890]{4})",
+    "Signatures": [
+      "JQQJ"
+    ],
+    "CreatedVersion": "1.18.0",
+    "LastUpdatedVersion": "1.18.0"
+  },
+  {
+    "Id": "SEC101/233",
+    "Name": "AzureDummyLegacyCommonAnnotatedSecurityKey",
```

Comment:
>AzureDummyLegacyCommonAnnotatedSecurityKey [](http://example.com/codeflow?start=13&length=42)

Cognitive services has reserved a key kind for the body of granular providers built on their platform. Some of these providers are for test purposes. Others have been deprecated or are no longer in use. We could investigate and attempt a more tuned inventory, all these checks certainly have test impact (notably! the net452 test execution is now double other CLR versions). OTOH, if the high performance regex caching is doing what it's supposed to, the overly broad rules definitions shouldn't have practical impact executing our API, yes?

Reply:
I think we're OK on perf now, but the name is bugging me. Can we use something else than 'dummy' here?

Reply:
That's the actual searchable name. :) Which argues for keeping it. I don't have a precise sense of what purpose this provider has, so I hesitate to finesse the language. I could ask about it if you feel strongly. ? What's your SEV on this, sir?